changehc/tests/test_download_ftp_files.py

```diff
@@ -51,22 +52,18 @@ def test_get_files(self, mock_path):
         get_files_from_dir(one_new_one_old, "00005566", "")
         assert one_new_one_old.num_gets == 1
 
-        # When seven new files are present, AssertionError
-        new_file1 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo1")
-        new_file2 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo2")
-        new_file3 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo3")
-        new_file4 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo4")
-        new_file5 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo5")
-        new_file6 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo6")
-        new_file7 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo7")
-        seven_new = self.MockSFTP([new_file1, new_file2, new_file3, new_file4,
-                                    new_file5, new_file6, new_file7])
+        # When too many new files are present, AssertionError
+        file_batch = [
+            self.FileAttr(dt.timestamp(dt.now()), f"00001122_foo{i}")
+            for i in range(EXPECTED_FILES_PER_DROP + 1)
```

Comment:
I think there should be `for i in range(1, EXPECTED_FILES_PER_DROP + 1)`

Reply:
Consider the following: if EXPECTED_FILES_PER_DROP=7, then for this test we want to construct a list of 8 files.

```
>>> len(list(range(7+1)))
8
>>> len(list(range(1, 7+1)))
7
```

Then we should use the former.

Reply:
You are right. Thanks.